model/src/main/scala/org/zalando/nakadi/client/model/Model.scala

```diff
@@ -0,0 +1,192 @@
+ * @param partitionResolutionStrategy Determines how the assignment of the event to a Partition should be handled.
+ * @param schema The schema for this EventType. This is expected to be a json schema in yaml format (other formats might be added in the future).
+ * @param dataKeyFields Indicators of the path of the properties that constitute the primary key (identifier) of the data within this Event. If set MUST be a valid required field as defined in the schema. (TBD should be required? Is applicable only to both Business and DataChange Events?)
+ * @param partitioningKeyFields Indicator of the field used for guaranteeing the ordering of Events of this type (used by the PartitionResolutionStrategy). If set MUST be a valid required field as defined in the schema.
+ *
+ */
+case class EventType(
+  name: String,
+  owner: String,
+  category: String,
+  effectiveSchema: String,
+  validationStrategies: Seq[String],
+  enrichmentStrategies: Seq[String],
+  partitionResolutionStrategy: PartitionResolutionStrategy, //Different naming
+  schema: EventTypeSchema,
+  dataKeyFields: Seq[String],
+  partitioningKeyFields: Seq[String])
+
+/**
+ * The schema for an EventType, expected to be a json schema in yaml format (other formats might be added in the future).
+ * @param type The type of schema definition (avro, json-schema, etc).
+ * @param schema The schema as string in the syntax defined in the field type. Failure to respect the syntax will fail any operation on an EventType.
+ */
+case class EventTypeSchema(
+  schemaType: String,
+  schema: String)
+
+/**
+ * Defines a rule for validation of an incoming Event. Rules might require additional parameters; see the `doc` field of the existing rules for details. See GET /registry/validation-strategies for a list of available rules.
+ * @param name Name of the strategy.
+ * @param doc Documentation for the validation.
+ */
+case class EventValidationStrategy(
+  name: String,
+  doc: String)
+
+/**
+ * Defines a rule for the resolution of incoming Events into partitions. Rules might require additional parameters; see the `doc` field of the existing rules for details. See `GET /registry/partition-strategies` for a list of available rules.
+ * @param name Name of the strategy.
+ * @param doc Documentation for the partition resolution.
+ */
+case class PartitionResolutionStrategy(
+  name: String,
+  doc: String)
+
+/**
+ * Defines a rule for transformation of an incoming Event. No existing fields might be modified. In practice this is used to set automatically values in the Event upon reception (i.e. set a reception timestamp on the Event). Rules might require additional parameters; see the `doc` field of the existing rules for details. See GET /registry/enrichment-strategies for a list of available rules.
+ * @param name Name of the strategy.
+ * @param doc Documentation for the enrichment.
+ */
+case class EventEnrichmentStrategy(
+  name: String,
+  doc: String)
+
```

Comment:
BatchItemResponse is missing
https://github.com/zalando/nakadi/blob/nakadi-jvm/api/nakadi-event-bus-api.yaml#L1318